STUDIO/org.openl.rules.repository.aws/src/org/openl/rules/repository/aws/S3Repository.java

```diff
@@ -402,10 +403,19 @@ private class S3RevisionGetter implements RevisionGetter {
 
         @Override
         public Object getRevision() {
-            FileData fileData;
             try {
-                fileData = check(MODIFICATION_FILE);
-                return fileData == null ? null : fileData.getVersion();
+                var fileData = check(MODIFICATION_FILE);
+                if (fileData == null) {
+                    log.error("Failed to detect changes. '{}' is not found.", MODIFICATION_FILE);

```

Comment:
This logging will happen very often, so move this messaging in the repo initialization block

Reply:
done